packages/web-forms/tests/components/common/map/createFeatureCollectionAndProps.test.ts

```diff
@@ -46,7 +46,7 @@ describe('createFeatureCollectionAndProps', () => {
 			],
 		});
 
-		expect(orderedExtraPropsMap).toEqual(new Map([['point1', []]]));
+		expect(orderedExtraPropsMap).toEqual(new Map());
```

Comment:
I must've missed something... what changed here?

Reply:
It removes properties that have no value, so they won't show up in the properties popup. This means less scrolling, too.